plugins/inputs/ssl/README.md

```diff
@@ -0,0 +1,35 @@
+# Example Input Plugin
+
+This input plugin will return how many days left for a SSL cert to expire.
+
+### Configuration:
+
+```
+# SSL request given a server, a Port, a timeout and a skipverify flag
+[[inputs.check_ssl]]
+  ## Server (default github.com)
+  servers = ["github.com"]
```

Comment:
I would recommend `servers = ["github.com:443"]` instead of a separate `port` param. This way you can have multiple ports without having to define a separate check.